### PR TITLE
MET-180: In Measurement Units, labels and symbol are not required

### PR DESCRIPTION
**Pull request by tseho**
**Description (for Contributor and Core Developer)**

When creating a new Measurement Family, labels and symbols are not required.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Akeneo/Tool/Bundle/MeasureBundle/spec/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidatorSpec.php [hunk 1]
```diff
@@ -75,4 +75,25 @@ function it_returns_an_empty_array_if_all_the_measurement_family_properties_are_
...
+        $measurementFamily = [
+            'code' => 'custom_metric_1',
+            'standard_unit_code' => 'CUSTOM_UNIT_1_1',
+            'units' => [
+                'CUSTOM_UNIT_1_1' => [
+                    'code' => 'CUSTOM_UNIT_1_1',
+                    'convert_from_standard' => [
+                        [
+                            'operator' => 'mul',
+                            'value' => '0.000001',
```
Comment (ValentinMumble) [inline src/Akeneo/Tool/Bundle/MeasureBundle/spec/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidatorSpec.php:90]: Shouldn't the standard operation be a multiply-by-one?

File: src/Akeneo/Tool/Bundle/MeasureBundle/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidator.php [hunk 2]
```diff
@@ -61,12 +60,14 @@ private function getJsonSchema(): array
...
-                            ]
+                            ],
+                            'required' => ['code', 'convert_from_standard'],
+                            'additionalProperties' => false,
                         ],
                     ]
                 ]
             ],
-            'required'             => ['code', 'labels', 'units', 'standard_unit_code'],
+            'required'             => ['code', 'units', 'standard_unit_code'],
```
Comment (juliensnz) [inline src/Akeneo/Tool/Bundle/MeasureBundle/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidator.php:70]: units are not mandatory for updates

**Comment by tseho** [comment]
What do you mean by "We should also change the unit validation IMHO" ?

File: src/Akeneo/Tool/Bundle/MeasureBundle/spec/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidatorSpec.php (hunk 1 above)
Comment (tseho) [reply src/Akeneo/Tool/Bundle/MeasureBundle/spec/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidatorSpec.php:90]: You are right, I'm fixing it but this is structure validation, not data validation :)

File: src/Akeneo/Tool/Bundle/MeasureBundle/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidator.php (hunk 2 above)
Comment (tseho) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Controller/ExternalApi/JsonSchema/MeasurementFamilyValidator.php:70]: On update, the validator is applied after the merge.

**Pull request merged**